static/assets/js/entity-reference/app.js

```diff
@@ -0,0 +1,59 @@
+(function(){
+    "use strict";
+
+    var app = angular.module('entityReferenceApp', []);
+
+    app.controller('EntityReferenceCtrl', ['$http', EntityReferenceCtrl]);
+
+    app.component('entityReference', {
+        templateUrl: '/assets/views/entities.html',
+        controller: 'EntityReferenceCtrl as ctrl',
+        bindings: {
+            title: '@',
+            getStartedUrl: '@',
+            swaggerUrl: '@',
+            whiteList: '@',
```

Comment:
Do you guys need the whiteList\blackList? I like the concept, but on our teams we're steering more towards updating the swagger to remove entities that we don't think should be displayed. To use the whitelist\blacklist, then we'd need to include entity names in the stache site, which I'd like to avoid if possible.